packages/snap-preact-components/src/components/Molecules/CalloutBadge/readme.md

```diff
@@ -31,7 +24,6 @@ The `componentMap` prop allows for custom badge components. This functionallity
 ```jsx
 <CalloutBadge 
     name={'callout'} 
-    controller={controller} 
     result={controller.store.results[0]} 
     componentMap={{
         'customOnSaleBadge': () => <div>On Sale</div>
```

Comment:
Component map details in readme should reflect the new changes (eg. async function/function that returns a component).